scene/resources/packed_scene.cpp

```diff
@@ -592,20 +592,26 @@ Node *SceneState::instantiate(GenEditState p_edit_state) const {
 		Callable callable(cto, snames[c.method]);
 		if (c.unbinds > 0) {
 			callable = callable.unbind(c.unbinds);
-		} else if (!c.binds.is_empty()) {
-			Vector<Variant> binds;
-			if (c.binds.size()) {
-				binds.resize(c.binds.size());
-				for (int j = 0; j < c.binds.size(); j++) {
-					binds.write[j] = props[c.binds[j]];
+		} else {
+			if (!c.binds.is_empty()) {
```

Comment:
This whole code can be improved.
You can add `Array binds`.
Then append `cfrom` to array if `CONNECT_APPEND_SOURCE_OBJECT` is enabled.
Then append the `c.binds`.
And finally if the binds array is not empty, use `bindv` on the callable.